### ISSUE TITLE
"Minecraft Class" helpers do not support Mojang's official mappings

### ISSUE THREAD

**Issue by ChloeDawn** [DemonWav/MinecraftDevIntelliJ#808]
**2020.2-1.5.2**

In a Fabric environment that uses official mappings, the helper attempts to use Yarn mappings.

```
mappings(minecraft.officialMojangMappings())
```

It should be using the mappings configured for the current workspace.

![Screenshot of new object tooltip](https://i.imgur.com/4UoeDCP.png)
![Screenshot of class creation modal](https://i.imgur.com/JUJr649.png)
![Screenshot of resulting class](https://i.imgur.com/c1ms0NG.png)

**Comment by Tom-The-Geek** [comment]
This was a feature I implemented, and I obviously didn't realise the mapping differences at the time. You should be able to adjust the templates if you want by going to Settings -> Editor -> File and Code Templates and then Skeletons under the Minecraft dropdown.

The reason it doesn't use the current mappings is that the template is predefined. I should probably work on it remapping the templates as it places them into a project or something similar.

**Comment by jamierocks** [comment]
Yeah, seems to me that the template should be in intermediary mappings and then resolved when created.

**Comment by DenWav** [comment]
Yeah, that's something we'll be able to do somewhat soon, once my overall mappings handling rewrite is done. Until then, editing the template is the best route.

**Comment by Earthcomputer** [comment]
Related to #771
